docs/.vitepress/theme/components/Feedback/Feedback.vue

```diff
@@ -14,16 +15,22 @@ enum FeedbackState {
 const route = useRoute();
 const currentState = ref(FeedbackState.START);
 const feedback = ref('');
+const feedbackTextarea = ref(null);
```

Comment:
```suggestion
const feedbackTextarea = ref<HTMLInputElement | null>(null);
```

I think you won't see the error with this!

(also I wanted to unresolved/resolved your previous comment on feedback but I probably unresolved/resolved some of other comments accidentally sorry 😢)

Reply:
🤦 🙏 ❤️